styles/new/base.css

```diff
@@ -0,0 +1,19 @@
+@tailwind base;
```

Comment:
These should not be moved to base.css

Reply:
@ovflowd There is an issue here, if I keep the `@tailwind base;` inside of the index.css there is a property that set the `<ul>` and `<ol>`  to `list-style: none;`, since the import is before the './base' file is not overriding the current default value making impossible to keep tha changes inside the base.css file.

The only way I could solve it its to keep the declaration inside of index.css but this goes agains this comment: https://github.com/nodejs/nodejs.org/issues/5935#issuecomment-1742123787, the question is: how should I proceed?

Reply:
importing `base` after `@tailwind base` will most probably fix this.

Reply:
When I implement the suggestion that you guys are proposing this is the undesirable output that I have, If I move the `@tailwind base;` as before works as expected.

![Screenshot from 2023-10-03 15-17-59](https://github.com/nodejs/nodejs.org/assets/6693254/cbdcf0f0-920f-449d-8753-e77286a7b29d)
![Screenshot from 2023-10-03 15-17-32](https://github.com/nodejs/nodejs.org/assets/6693254/905333cf-6ea5-4714-a4eb-2b003e16c274)

Reply:
And also the StyleLint triggers an error: `Unexpected invalid position @import rule (no-invalid-position-at-import-rule)Stylelint[no-invalid-position-at-import-rule](https://stylelint.io/user-guide/rules/no-invalid-position-at-import-rule)`

Reply:
That's because you should move both imports after `@tailwind`

Reply:
I'm 100% sure that just moving the imports after `@tailwind` solves your issue, that's essential CSS. CSS follows rules based on a set of priorities. Also, another PR faced the same, and they fixed it by doing this. So you might want to investigate further.

Reply:
I put both as you said, the output is wrong and the linter is blocking me.
![Screenshot from 2023-10-03 15-59-58](https://github.com/nodejs/nodejs.org/assets/6693254/566cadcd-e771-4a8b-b4a2-d1d146f7797d)

Even if I try to force the linter rule the style on the tailwind base persists.
![Screenshot from 2023-10-03 16-01-52](https://github.com/nodejs/nodejs.org/assets/6693254/b8a57161-b391-4f67-bc9b-161dcbe2ba50)

Reply:
Interesting, I've identified the issue. And pushed a hotfix. Can you rebase your branch?

Reply:
@ovflowd what is the difference between `@import tailwind/` and `@tailwind`?

Reply:
`@tailwind` hooks directly into PostCSS engine, as `@tailwind` is a custom PostCSS plugin, this usually has faster loading and building of CSS and has some optimizations.

But for us at least the performance impact isn't noticeable. The regular import also imports the whole Tailwind styles instead of invoking Tailwind's plugin to simply add what we need (it prescans the codebase); The Tailwind PostCSS plugin will still do transformations and afterwards remove what we don't use (with regular imports)